.github/actions/gpu_setup/action.yml

```diff
@@ -74,6 +77,11 @@ runs:
       env:
         GCC_VERSION: ${{ inputs.gcc-version }}
 
+    - name: Check setup
```

Comment:
check we actually set up CUDA correctly so we don't run all the other steps if they are bound to fail